### PR TITLE
created and styled messageBox component to be used multiple times

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: js/components/NoteList/AddNote/styles.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const styles = StyleSheet.create({
     marginTop: 22,
   },
   container: {
-    top: height / 3,
+    top: height / 3.7,
```
Comment (siddharthparmar7) [inline js/components/NoteList/AddNote/styles.js]: do not use height and width for margin, padding or top/bottom/left/right. Thank you

File: js/components/NoteList/AddNote/AddNote.js [hunk 2]
```diff
@@ -21,28 +25,14 @@ export default class AddNote extends Component {
...
-              />
-              <TouchableOpacity activeOpacity={0.5} style={styles.button}>
-                <Text style={styles.submit}>Submit</Text>
-              </TouchableOpacity>
-            </View>
+            <MessageBox
+              close={close}
+              updateText={text => this._updateText(text)}
+              showCloseIcon
+              value={this.state.text}
```
Comment (siddharthparmar7) [inline js/components/NoteList/AddNote/AddNote.js]: the final form takes care of this. so you do not need to pass this as a prop and have a state for that.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by pjchong91** [comment]
A new component is to be made rather than resuing the MessageBox component.

**Pull request closed without merging**